lib/saml/user.rb

```diff
@@ -79,7 +80,17 @@ def authn_context
       raise
     end
 
+    # For future use; can check if issuer matches /eauth\.va\.gov/ for SSOe
+    def issuer
+      saml_response.issuers[0]
+    rescue StandardError
```

Comment:
rubocop was just upgraded in master and now wants the just `rescue` with `StandardError` being implicit